bootstrap.py

```diff
@@ -918,11 +930,49 @@ class MEOptions:
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule and options.foreman_fqdn:
+        # > ELIF new_capsule and foreman_fqdn set, will migrate to other capsule
+        #
+        # > will replace CA certificate, reinstall katello-agent, gofer
+        # > update system definition to point to new capsule for content,
+        # > Puppet, OpenSCAP and update Puppet configuration
+        # > MANUAL SIGNING OF CSR OR MANUALLY CREATING AUTO-SIGN RULE STILL REQUIRED!
+        # > API doesn't have a public provision for creating auto-sign entries yet!
+        get_bootstrap_rpm(clean=True)
+        install_katello_agent()
+        API_PORT = get_api_port()
+
+        print_running("Calling Foreman API to update content source, puppet master, puppet ca and openscap proxy records for %s" % FQDN)
+        capsule_id = return_matching_foreman_key('smart_proxies', 'name="%s"' % options.foreman_fqdn, 'id', False)
+        host_id = return_matching_foreman_key('hosts', 'name="%s"' % FQDN, 'id', False)
+        update_host_capsule_mapping("puppet_proxy_id", capsule_id, host_id)
+        update_host_capsule_mapping("puppet_ca_proxy_id", capsule_id, host_id)
+        update_host_capsule_mapping("content_source_id", capsule_id, host_id)
+        update_host_capsule_mapping("openscap_proxy_id", capsule_id, host_id)
```

Comment:
It is not guaranteed that every capsule runs the OpenSCAP feature. In testing, migrating from one Capsule to another (where the destination did not have the OpenSCAP feature), bootstrap.py fails with an error message similar to

~~~
RUNNING], [2017-03-30 06:22:12], [Calling Foreman API to update content source, puppet master, puppet ca and openscap proxy records for devnode-115.example.com] 
An error occured: HTTP Error 422: Unprocessable Entity
url: https://capsule-2.example.com:8443/api/v2/hosts/231
code: 422
data: {
  "host": {
    "openscap_proxy_id": "3"
  }
}
error: {
  "error": {
    "errors": {
      "openscap_proxy_id": [
        "must have Openscap feature"
      ]
    }, 
    "id": 231, 
    "full_messages": [
      "Openscap Capsule must have Openscap feature"
    ]
  }
}

~~~

It may make sense to check the Capsule's features, and then update the host if (and only if) it has the OpenSCAP feature.